state/modelconfig.go

```diff
@@ -219,7 +219,7 @@ func (st *State) buildAndValidateModelConfig(updateAttrs attrValues, removeAttrs
 	if err := checkModelConfig(newConfig); err != nil {
 		return nil, errors.Trace(err)
 	}
-	return st.validate(newConfig, oldConfig)
+	return newConfig, nil
```

Comment:
Any reason we're not validating this any more?

Reply:
Model config validation is handled by the model domain - this code will be removed in [JUJU-5343](https://warthogs.atlassian.net/browse/JUJU-5343)